docquality.rst

```diff
@@ -76,9 +80,10 @@ Helping with the Developer's Guide
 
 .. highlight:: console
 
-The Developer's Guide uses the same process as the main Python documentation,
-except for some small differences.  The source lives in a `separate
-repository`_ and bug reports should be submitted to the `the GitHub tracker`_.
+The Developer's Guide (what you're reading now) uses the same process as the
+main Python documentation, except for some small differences.  The source
+lives in a `separate repository`_ and bug reports should be submitted to
+the `the GitHub tracker`_.
```

Comment:
"it's `GitHub tracker`_` (basically helping differentiate which GitHub issue tracker you're talking about and dropping the double "the").

Reply:
"its"

Reply:
Oops. 😄 Updated my comment according to Ned's catch of my grammatical mistake.

Reply:
Done.